tendrl/ceph_integration/manager/request_factory.py

```diff
@@ -20,6 +20,9 @@ def __init__(self, cluster_monitor):
     def delete(self, obj_id):
         raise NotImplementedError()
 
+    def delete_rbd(self, pool_id, name):
```

Comment:
As commented in https://github.com/Tendrl/ceph-integration/pull/108#discussion_r100976810